action.yaml

```diff
@@ -34,6 +34,10 @@ inputs:
     description: 'If true, only check if the test scripts are valid and skip test execution'
     default: "false"
     required: false
+  show-k6-progress-output:
```

Comment:
Would `debug` be a better name for this option?

Reply:
That makes sense, I'll update it. We can reuse it in other places as well.

Reply:
Done